pg_replicate/src/clients/delta.rs

```diff
@@ -36,13 +125,18 @@ impl DeltaClient {
             &Type::CHAR | &Type::BPCHAR | &Type::VARCHAR | &Type::NAME | &Type::TEXT => {
                 DataType::STRING
             }
-            &Type::INT2 | &Type::INT4 | &Type::INT8 => DataType::INTEGER,
-            &Type::FLOAT4 | &Type::FLOAT8 | &Type::NUMERIC => DataType::FLOAT,
+            &Type::INT4 => DataType::INTEGER,
+            &Type::INT8 => DataType::LONG,
+            &Type::INT2 => DataType::SHORT,
+            &Type::FLOAT4 => DataType::FLOAT,
+            &Type::FLOAT8 | &Type::NUMERIC => DataType::DOUBLE,
```

Comment:
I don't think PG's Numeric type will fit in a double. It should be a decimal type.

Reply:
do you know the precission?